code/api/src/modules/userProduct/model.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
This is interesting that this auto-generated file with `sequelize` codes the file differently than the original files in the app.